00-HTML-CSS-basics/layout2.6.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta content="name=value,name=value" name="viewport">
+    <meta charset="utf-8">
+    <link href="layout2.6.css" rel="stylesheet" type="text/css">
+    <title>Layout</title>
+</head>
+<body>
+
+<!-- BUG: when the mouse pointer is over the text, the image stays in black and white -->
```

Comment:
I based on the picture to do the layout.htm, then I saw the actual page with which we had to guide on to do the layout and I saw it had links in the words in bold of the header, in the pictures and in the Title and description of the pics. So, I added all those details in the layout2.6_html. But yes, they are pretty similar.